roles/splunk_heavy_forwarder/molecule/default/converge.yml

```diff
@@ -1,82 +1,5 @@
 ---
 - name: Converge
   hosts: all
-  vars:
```

Comment:
Whats the motivation behind moving these vars over to `/molecule/default/molecule.yml`?

Reply:
@arctan5x noticed a while ago that when variables were overwritten in any of the plays, there were some negative consequences. For instance, if some playbook had:
```
- set_fact:
    splunk:
        build_location: /path/to/new/splunk.tgz
```
what ended up happening was the entire `splunk` dict got replaced so that it becomes:
```
{"splunk": {"build_location": "/path/to/new/splunk.tgz"}}
```

This mean things like `splunk.home`, `splunk.user` were undefined afterwards. This only happens during the `molecule` execution, so we suspect it had to do with Ansible's [variable order-of-precedence](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable).

I haven't confirmed if the var movement here fixes this problem, but I was hoping to address that issue Tony found with the test refactoring work.